redash/authentication/__init__.py

```diff
@@ -41,6 +41,10 @@ def sign(key, path, expires):
 
 @login_manager.user_loader
 def load_user(user_id_with_identity):
+    user = api_key_load_user_from_request(request)
+    if user:
+        return user
```

Comment:
This feels almost as too simple 😆

Reply:
I've battled flask_login for 8 hours, trying to get it to cooperate with `user_loader` and `request_loader` until this hit me 🤦‍♂️